sympy/parsing/autolev/test-examples/ruletest9.py

```diff
@@ -1,53 +1,53 @@
-a=0
-d=me.inertia(frame_a, 1, 1, 1)
-point_po1 = me.Point('po1')
-point_po2 = me.Point('po2')
-particle_p1 = me.Particle('p1', me.Point('p1_pt'), sm.Symbol('m'))
-particle_p2 = me.Particle('p2', me.Point('p2_pt'), sm.Symbol('m'))
-c1, c2, c3 = me.dynamicsymbols('c1 c2 c3')
-c1d, c2d, c3d = me.dynamicsymbols('c1 c2 c3', 1)
-body_r_cm = me.Point('r_cm')
+frame_n = _me.ReferenceFrame('n')
+frame_a = _me.ReferenceFrame('a')
+a = 0
+d = _me.inertia(frame_a, 1, 1, 1)
+point_po1 = _me.Point('po1')
+point_po2 = _me.Point('po2')
+particle_p1 = _me.Particle('p1', _me.Point('p1_pt'), _sm.Symbol('m'))
+particle_p2 = _me.Particle('p2', _me.Point('p2_pt'), _sm.Symbol('m'))
+c1, c2, c3 = _me.dynamicsymbols('c1 c2 c3')
+c1_d, c2_d, c3_d = _me.dynamicsymbols('c1 c2 c3', 1)
+body_r_cm = _me.Point('r_cm')
 body_r_cm.set_vel(frame_n, 0)
-body_r_f = me.ReferenceFrame('r_f')
-body_r = me.RigidBody('r', body_r_cm, body_r_f, sm.symbols('m'), (me.outer(body_r_f.x,body_r_f.x),body_r_cm))
+body_r_f = _me.ReferenceFrame('r_f')
+body_r = _me.RigidBody('r', body_r_cm, body_r_f, _sm.symbols('m'), (_me.outer(body_r_f.x, body_r_f.x), body_r_cm))
 point_po2.set_pos(particle_p1.point, c1*frame_a.x)
-v=2*point_po2.pos_from(particle_p1.point)+c2*frame_a.y
+v = 2*point_po2.pos_from(particle_p1.point)+c2*frame_a.y
 frame_a.set_ang_vel(frame_n, c3*frame_a.z)
-v=2*frame_a.ang_vel_in(frame_n)+c2*frame_a.y
+v = 2*frame_a.ang_vel_in(frame_n)+c2*frame_a.y
 body_r_f.set_ang_vel(frame_n, c3*frame_a.z)
-v=2*body_r_f.ang_vel_in(frame_n)+c2*frame_a.y
+v = 2*body_r_f.ang_vel_in(frame_n)+c2*frame_a.y
 frame_a.set_ang_acc(frame_n, (frame_a.ang_vel_in(frame_n)).dt(frame_a))
-v=2*frame_a.ang_acc_in(frame_n)+c2*frame_a.y
+v = 2*frame_a.ang_acc_in(frame_n)+c2*frame_a.y
 particle_p1.point.set_vel(frame_a, c1*frame_a.x+c3*frame_a.y)
 body_r_cm.set_acc(frame_n, c2*frame_a.y)
-v_a = me.cross(body_r_cm.acc(frame_n), particle_p1.point.vel(frame_a))
+v_a = _me.cross(body_r_cm.acc(frame_n), particle_p1.point.vel(frame_a))
 x_b_c = v_a
 x_b_d = 2*x_b_c
-a_b_c_d_e=x_b_d*2
+a_b_c_d_e = x_b_d*2
 a_b_c = 2*c1*c2*c3
 a_b_c += 2*c1
-a_b_c  =  3*c1
-q1, q2, u1, u2 = me.dynamicsymbols('q1 q2 u1 u2')
-q1d, q2d, u1d, u2d = me.dynamicsymbols('q1 q2 u1 u2', 1)
-x, y = me.dynamicsymbols('x y')
-xd, yd = me.dynamicsymbols('x y', 1)
-xd2, yd2 = me.dynamicsymbols('x y', 2)
-yy = me.dynamicsymbols('yy')
-yy = x*xd**2+1
-m = sm.Matrix([[0]])
+a_b_c = 3*c1
+q1, q2, u1, u2 = _me.dynamicsymbols('q1 q2 u1 u2')
+q1_d, q2_d, u1_d, u2_d = _me.dynamicsymbols('q1 q2 u1 u2', 1)
+x, y = _me.dynamicsymbols('x y')
+x_d, y_d = _me.dynamicsymbols('x y', 1)
+x_dd, y_dd = _me.dynamicsymbols('x y', 2)
+yy = _me.dynamicsymbols('yy')
+yy = x*x_d**2+1
+m = _sm.Matrix([[0]])
 m[0] = 2*x
-m = m.row_insert(m.shape[0], sm.Matrix([[0]]))
+m = m.row_insert(m.shape[0], _sm.Matrix([[0]]))
 m[m.shape[0]-1] = 2*y
 a = 2*m[0]
-m = sm.Matrix([1,2,3,4,5,6,7,8,9]).reshape(3, 3)
-m[0,1]=5
+m = _sm.Matrix([1,2,3,4,5,6,7,8,9]).reshape(3, 3)
+m[0,1] = 5
```

Comment:
Please note that these files are just the outputs generated by the parser. Changes made here are not useful. The write statements in the parser code in `_listener_autolev_antlr.py` are the ones which should be changed.
@moorepants Seems like people are confused by this. Is there any way to make it clear that these are just example codes? Maybe use .txt files instead of .py?

Reply:
The write statements  in `_listerner_autolve_antlr.py` seems to be properly formatted to give spaces around the `=` .Should I keep these changes in `ruletest9.py` or revert back ?

Reply:
@ayushmankoul 
You will have to take a closer look. The examples wouldn't turn out like this if that was the case. Also, you shouldn't worry about making changes to the example files at all. You just have to update the parser code and run the script to update all the examples before commiting.

Reply:
@NikhilPappu Sir I have updated the test examples.Please have a review of it.